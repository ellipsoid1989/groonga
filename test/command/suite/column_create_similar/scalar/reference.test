table_create Tags TABLE_PAT_KEY ShortText

table_create Notes TABLE_NO_KEY
column_create Notes tags COLUMN_SCALAR Tags

table_create NotesSimilar TABLE_NO_KEY
column_create_similar NotesSimilar tags Notes.tags

dump
