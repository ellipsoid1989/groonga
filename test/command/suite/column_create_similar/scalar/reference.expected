table_create Tags TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
table_create Notes TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Notes tags COLUMN_SCALAR Tags
[[0,0.0,0.0],true]
table_create NotesSimilar TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create_similar NotesSimilar tags Notes.tags
[[0,0.0,0.0],true]
dump
table_create Notes TABLE_NO_KEY

table_create NotesSimilar TABLE_NO_KEY

table_create Tags TABLE_PAT_KEY ShortText

column_create Notes tags COLUMN_SCALAR Tags
column_create NotesSimilar tags COLUMN_SCALAR Tags
